demos/README.md

```diff
@@ -47,7 +47,7 @@ There are several variables you will want to set that are listed at the top of t
 
 ## Buildah and Docker compatibility demo
 
-filename: [`buildah-scratch-demo.sh`](https://github.com/projectatomic/buildah/demos/docker-compatibility-demo.sh)
+filename: [`docker-compatibility-demo.sh`](https://github.com/projectatomic/buildah/blob/masterdemos/docker-compatibility-demo.sh)
```

Comment:
I think you want a slash between the master and demos like you have in line 32.  Besides that, could you do a relative link instead?  (demos/docker-compatibity-demo.sh)